src/f4/app_blinky/main.c

```diff
@@ -31,6 +31,12 @@ int main(void)
     chSysInit();
 
     palSetLineMode(LINE_LED,PAL_MODE_OUTPUT_PUSHPULL);
+#if defined STM32F439xx
```

Comment:
This stuff should be part of the `board.h` file. Otherwise this is going to be applied to any other boards (like a C3) that someone might be testing this app on.